homeassistant/components/media_player/russound_rnet.py

```diff
@@ -91,33 +93,54 @@ def name(self):
 
     @property
     def state(self):
-        """Return the state of the device."""
-        return self._state
+        """Return the state of the device. directly from the device."""
+        if self._russ.get_power('1', self._zone_id) == 0:
```

Comment:
Very useful info.  Thank you.  The NAD.py is a great example as its simple to follow, but demonstrates the correct implementation.
I have implemented the changes.